onnxruntime/contrib_ops/cpu/skip_layer_norm.cc

```diff
@@ -290,16 +257,27 @@ template <typename T, bool simplified>
 Status SkipLayerNorm<T, simplified>::PrePack(const Tensor& tensor, int input_idx, AllocatorPtr alloc,
                                              bool& is_packed, PrePackedWeights* prepacked_weights) {
   ORT_UNUSED_PARAMETER(prepacked_weights);
+  ORT_UNUSED_PARAMETER(tensor);
```

Comment:
it looks like the `tensor`, `input_idx` and `alloc` params are used below - why is `ORT_UNUSED_PARAMETER` needed for them in that case?

Reply:
I was trying to see is issue is in prepacking by disable it. Forget to undo it